src/app.ts

```diff
@@ -26,6 +26,16 @@ export const initDb = async (): Promise<DatabaseManager> => {
 const app: Application = express();
 const config = appConfig();
 
+// DO NOT REMOVE!
+// DuckDB handles numbers as bigints.  BigInts don't serialise
+// toJSON easily.  This monkypatches BigInt so that if the number
+// is less than the max safe interger we return a number otherwise
+// we return a string
+(BigInt.prototype as any).toJSON = function () {
+    if (this < Number.MAX_SAFE_INTEGER) return Number(this);
+    return this.toString();
+};
+
```

Comment:
I'd probably stick this in it's own file and import it, but not a blocker.

Reply:
Lets refactor later... Right now I think this PR hangs together with shoestring and sellotape and I don't want to change it to much 😆